serverless/serverless-getting-started.adoc

```diff
@@ -20,8 +20,16 @@ For information about installing {ServerlessProductName}, see xref:../serverless
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
+[id="knative-eventing_{context}"]
+== Knative Eventing
```

Comment:
@aheslin, does this language sound fine to you?

Reply:
@sferich888, are you ok with this GitHub.io docs link in this context?

Reply:
In this context it's fine, as we get out of Dev Preview and Technology preview we should remove this.

Reply:
@abrennan89, please plan to bring this content into the collection when it enters tech preview and remove this link.